docs/en/FACL.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: global
+title: Access Control List
+nickname: Access Control List
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+This document describes the Access Control List feature related to file and directory permissions in Alluxio. 
+
+1. [ACL Permission Model](#ACL Permission Model): Alluxio implements an ACL model similar to those found in Linux and HDFS. This section describes some background on what this permission model looks like. 
+1. [Command Line Interface](#Command Line Interface): The users can interact with Alluxio's ACL model by using the command line interface tools such as `setfacl` and `getfacl`. 
+
+## ACL Permission Model
+Alluxio already implements a file and directory permission model similar to traditional Unix permission bits. For each directory and file, there are three types of permissions: read, write and execute. These permissions can be assigned to three different user clases: owner, group and other. 
```

Comment:
is this talking about the old permission model that was replaced by ACLs, or do users decide which model they want?

I think the docs should focus on how things work now - discussing how things worked historically might confuse the user about how things work today.